### DISCUSSION TITLE
May I ask if this driver supports the RTL8157 5Gbps network card?

### DISCUSSION THREAD

**Discussion by tltangliang** [bb-qq/r8152#417] [Q&A] [answered]
I purchased the WisdPi USB 5G network card, which is not compatible with QNAP NAS. I would like to inquire if it is supported on Synology NAS. If it is supported, I may consider purchasing a Synology NAS.

**Comment by bb-qq** [comment, marked as answer]
Some users reported it does work. #428

**Reply by tltangliang** [reply]
That's great.
